### ISSUE TITLE
Adding custom databases

### ISSUE THREAD

**Issue by pck00** [gbouras13/pharokka#249]
Hey, I'm trying to add a custom database to pharokka, but simply adding an mmseqs2 database to the database folder doesn't work (looks like there's specific code to call individual databases sadly).

I was wondering if there was an easy way to add custom databases to pharokka like in prokka, or if such a functionality is planned. 

cheers

**Comment by gbouras13** [comment]
Hi @pck00,

Unfortunately there is no way to add custom DBs in Pharokka right now. Pharokka developed out of some scripts I had been using that focused on Phanotate and PHROGs, so the computer science is not the best (I'd code it differently now if I wrote it from scratch!) - a lot of the processing code is hard coded. I also decided to make some of the output PHROGs centric (i.e. functional groups in the cds_functions.tsv and the new plotting function).

However, I will aim to add some custom DB functionality for the next major update when I find some time - I imagine that this might be a more useful feature especially as more phage research is done and the databases explode in size.

George
